app/views/test/index.html.erb

```diff
@@ -1,2 +1,212 @@
-<p>This is a test page. It will serve as the home page for logged in users.</p>
-<p>In the meantime, however, have a Weibo alien. <i class="fa fa-weibo"></i></p>
+<div class="col-md-8" style="padding:0px;">
+  <div class="panel panel-default" style="border-radius:15px;margin-left:10px;margin-right:10px;">
```

Comment:
This is not the right way to customize UI elements. I don't agree with modifying the Bootstrap theme at all, but if necessary then they should be in a separate CSS file that is automatically loaded in the assets pipeline.